shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -1032,6 +1028,12 @@ public boolean shouldRestoreAndSaveState() {
     return true;
   }
 
+  @Override
+  public boolean popSystemNavigator() {
```

Comment:
Is this also available with  `FlutterActivity.createDefaultIntent(currentActivity)` and `FlutterFragment.createDefault()` ? if not maybe we should provide `PlatformPluginDelegate setter` something like flutterActivity.setPlatformPluginDelegate(new PlatformPluginDelegate).

Reply:
Generally, we prefer users to subclass then use one of the constructors like NewEngineIntentBuilder, CachedEngineIntentBuilder, NewEngineFragmentBuilder etc to build the subclass.

Reply:
Great work